topic 5/src/main/java/topic5/StudentDAO.java

```diff
@@ -0,0 +1,18 @@
+package topic5;
+
+import org.mongodb.morphia.Morphia;
+import org.mongodb.morphia.dao.BasicDAO;
+
+import com.mongodb.Mongo;
+import com.mongodb.MongoClient;
+ 
+
+ 
+	public class StudentDAO extends BasicDAO<Student,String> {
+
+		protected StudentDAO( MongoClient mongoClient, Morphia morphia,  String dbName) {
+			super(mongoClient, morphia,dbName);
+			// TODO Auto-generated constructor stub
```

Comment:
Always remove TODO comments if you won't need to mark that you will code something in future